template/AssetName.bat

```diff
@@ -6,13 +6,15 @@
 @echo off
 call _mkasset %~n0 %~dp0%
 
-:: Following is the variables that can be set in the bat
-::
-:: They are default set to "nothing"
-:: leave them be if you dont have the info for them and return to them later.
-:: setOBJECT expects a string like "prop, char"
-:: MINDBENDER_EDIT_IN and MINDBENDER_EDIT_OUT sets the assets time in and out for the edit. 
-:: These should not be set for assets, use this options for shots
-set OBJECT3D=
-set MINDBENDER_EDIT_IN=
-set MINDBENDER_EDIT_OUT=
\ No newline at end of file
+:: ------------------
+:: Additional variables
+:: ------------------
+
+:: (str, optional): This variable is ??????, for example "prop, char"
+set MINDBENDER_OBJECT3D=
```

Comment:
One thing before we can merge this, what is this variable? I couldn't find it referenced anywhere in the code.